content/release-notes/1.19.2.md

```diff
@@ -0,0 +1,12 @@
+---
+date: 2020-09-17
+linktitle: "1.19.2"
+title: "1.19.2"
+kubernetes: "1.16, 1.17, and 1.18"
+---
+
+{{<fixes>}}
+* Fixed a bug that could cause the master node join script to fail to be created on the Cluster Management tab.
```

Comment:
~master~ primary

Reply:
while I agree, the page currently says master:
![image](https://user-images.githubusercontent.com/2318911/93520587-920fe000-f8e3-11ea-9907-9999fd67b446.png)